src/plugins/memory/memory.yml

```diff
@@ -10,3 +10,12 @@ tileA:
 tileB:
   en: Card B
   de: Karte B
+text:
+  en: text
```

Comment:
The other labels in the editor (like "Width", "Card pairs", etc.) are capitalized also in english. Let's also adjust "text" and "media URL" ("Text" already exists in `common.yml`)

Reply:
Indeed, streamlined them, thanks for noticing.